docs/src/index.js

```diff
@@ -2,11 +2,16 @@ import '@babel/polyfill'
 
 import React from 'react'
 import ReactDOM from 'react-dom'
+import faker from 'faker'
 
 import App from './App'
 
 export default App
 
+if (process.env.FAKER_SEED) {
+  faker.seed(parseInt(process.env.FAKER_SEED, 10))
```

Comment:
This is so that the "random" values are deterministic for each run of the test suite. The normal docs outside of running the visual tests still won't use a seed because it's not provided in that case.